### ISSUE TITLE
Google OptimizationGuidePredictionModels file internal downloads in the background

### ISSUE THREAD

**Issue by pksignX** [ultrafunkamsterdam/undetected-chromedriver#1293]
While loading the driver it automatically downloads a set of file with client name OptimizationGuidePredictionModels, which consumes around 9MB of network bandwith. Is there any way to avoid it?

URL: chrome://download-internals/
[Download list image](https://drive.google.com/file/d/1vPepIvpfi6St2EKrlSQySwcxEUPnG8lz/view?usp=sharing)

**Comment by Redrrx** [comment]
use this argument during launch of the driver or chroem and let us know how it goes for you, this issue isn't really related to UCD

```
--disable-features=OptimizationGuideModelDownloading,OptimizationHintsFetching,OptimizationTargetPrediction,OptimizationHints
```

**Comment by pksignX** [comment]
Yes it worked. It stopped downloading OptimizationGuidePredictionModels. 

But still i see a network usage of 9 MB every time i load the driver object. Not sure what it is downloading.

**Comment by pksignX** [comment]
This is a sample code:

`import psutil
import undetected_chromedriver as uc

io = psutil.net_io_counters()
bytes_sent, bytes_recv = io.bytes_sent, io.bytes_recv
driver = uc.Chrome(version_main=110)

io_2 = psutil.net_io_counters()
us, ds = io_2.bytes_sent - bytes_sent, io_2.bytes_recv - bytes_recv
print("uploaded bytes {} MB ".format(us / 1024 / 1024))
print("downloaded bytes {} MB ".format(ds / 1024 / 1024))`


output:
uploaded bytes 0.068359375 MB 
downloaded bytes 9.427734375 MB
